Backpack/src/main/java/net/skyscanner/backpack/checkbox/BpkCheckbox.kt

```diff
@@ -52,7 +52,7 @@ open class BpkCheckbox @JvmOverloads constructor(
     var checkmarkDefaultColor = context.getColor(R.color.bpkTextSecondary)
     var checkmarkCheckedColor = BpkTheme.getPrimaryColor(context)
     var checkmarkDisabledColor = context.getColor(R.color.bpkSkyGrayTint04)
-    val textDisabledColor = context.getColor(R.color.bpkSkyGrayTint04)
+    val textDisabledColor = context.getColor(R.color.__textDisabled)
```

Comment:
created a generic disabled text colour as we need it lots. ideally we'd actually create this as a token, but we might just wanna do that as part of the colour refresh when we revisit all our colour tokens, rather than now